dashboard/client/src/pages/job/JobDetail.tsx

```diff
@@ -104,8 +119,19 @@ export const JobDetailChartsPage = () => {
     <div className={classes.root}>
       <JobMetadataSection job={job} />
 
+      {data?.datasets && data.datasets.length > 0 && (
+        <CollapsibleSection
+          title="Ray Data Overview"
+          className={classes.section}
+        >
+          <Section>
+            <DataOverview datasets={data.datasets} />
+          </Section>
+        </CollapsibleSection>
+      )}
+
       <CollapsibleSection
-        title="Tasks/actor overview (beta)"
+        title="Ray Core Overview"
```

Comment:
is this related to the other code changes / intended?

Reply:
Just a change @scottsun94 wanted, to make a clear distinction between what table ray data/core users should be looking at